playbooks/openshift-web-console/private/config.yml

```diff
@@ -13,10 +13,13 @@
 
 - name: Web Console
   hosts: oo_first_master
-  roles:
-  - openshift_web_console
   vars:
     first_master: "{{ groups.oo_first_master[0] }}"
+  tasks:
+  - debug: msg="{{ openshift_version | version_compare('3.9', '>=') }}"
+  - import_role:
+      name: openshift_web_console
+    when: openshift_version | version_compare('3.9', '>=')
```

Comment:
Why the preference to do the version check here rather than just updating the check in its existing location?

Reply:
Nevermind, your PR description provided justification. Though I didn't understand that to be the case. I thought static import_playbook would skip it entirely in a static manner.